### ISSUE TITLE
tq_index("SP500") - Error NA names 

### ISSUE THREAD

**Issue by mdancho84** [business-science/tidyquant#142]
This is to address a recent issue with changes to SPDRS data format.
`tq_index()` - Fix naming issue with stock index data downloaded from www.us.spdrs.com

**Comment by mdancho84** [comment]
Fixed with v0.5.8

**Comment by zac-garland** [comment]
I'm running into an error during index download on v0.5.8.

I think they changed the link. If you visit the link that `spdr_link` outputs, it hits a redirect page and ends up downloading an HTML page into an excel file. 

I can download the second link however. Would you check if it's working on your end?

```
# current download method
x <- "SPY"
spdr_link <- paste0("https://us.spdrs.com/site-content/xls/", x, "_All_Holdings.xls")
httr::GET(spdr_link, httr::write_disk(tf <- tempfile(fileext = ".xlsx")))
readxl::read_xlsx(tf, skip = 3)

# working link
spdr_link <- "https://www.ssga.com/us/en/individual/etfs/library-content/products/fund-data/etfs/us/holdings-daily-us-en-spy.xlsx"
httr::GET(spdr_link,httr::write_disk(tf <-  tempfile(fileext = ".html")))
readxl::read_xlsx(tf, skip = 3)

```
